### DISCUSSION TITLE
Can any HLS player support LL-HLS playback?

### DISCUSSION THREAD

**Discussion by tremzo** [ant-media/Ant-Media-Server#5660] [FAQs]
The application settings have a checkbox to "Enable Low Latency HLS Streaming(CMAF)", can I use my standard HLS player to support low latency HLS or do I need a specific player to support LL-HLS?

**Comment by mekya** [comment]
Hi @timantmedia,

Thank you for asking the question. Let me try to answer.

Standard HLS players likely cannot play low-latency HLS because it has different dynamics. You just need to make sure that your player supports LL-HLS.

Additionally, there is a difference between Apple LL-HLS and CMAF HLS streaming. Ant Media Server supports both low-latency HLS and low-latency Dash. They use the CMAF behind the scenes. The Ant Media Server embedded player supports playing low-latency DASH but not low-latency HLS for now.

Please let us know if you have further questions

Regards
Oguz
